src/scene/forward-renderer.js

```diff
@@ -174,7 +174,7 @@ pc.extend(pc, function () {
         small.size = 0;
         large.size = 0;
 
-        // Grouping vertices according to the position related the the face
+        // Grouping vertices according to the position related the face

```

Comment:
This should be 'Grouping vertices according to the position related to the face'